src/main/java/seedu/address/model/account/Transaction.java

```diff
@@ -0,0 +1,12 @@
+package seedu.address.model.account;
+
+/**
+ * Temporary transaction class
+ */
+public class Transaction {
```

Comment:
I think you can shift this class into the right package like where Daryl has placed his, so that when merging, it will be smoother.

Reply:
I think it would be ok because it would be deleted after merging. Besides, daryl's branch is not merged yet, so there's no package to put it.

Reply:
Jiehan, if you want to keep this temp file for now, perhaps you could place it into ```model/account/stub```, so we can be sure to delete the correct Transaction class afterwards.

Reply:
I meant it's currently in your account package. It should be on it's own package so when merging, we see the conflict and can resolve accordingly